doc/developer/design/20230705_v2_txn_management.md

```diff
@@ -321,19 +321,21 @@ Reads of data shards are almost as straightforward as writes. A data shard may
 be read normally, using snapshots, subscriptions, shard_source, etc, through the
 most recent non-empty write. However, the upper of the txns shard (and thus the
 logical upper of the data shard) may be arbitrarily far ahead of the physical
-upper of the data shard. As a result, we have to "translate" timestamps by
-synthesizing ranges of empty time that have not yet been committed (but are
-guaranteed to remain empty):
+upper of the data shard. As a result, we do the following:
 
 - To take a snapshot of a data shard, the `as_of` is passed through unchanged if
```

Comment:
Reminder to self: update the rustdoc in the txns crate with this text once we hash out the wording in code review